src/main/java/org/folio/inventory/support/ItemUtil.java

```diff
@@ -192,8 +193,7 @@ public static Item jsonToItem(JsonObject itemRequest) {
     List<String> formerIds = toListOfStrings(
       itemRequest.getJsonArray(Item.FORMER_IDS_KEY));
 
-    List<String> statisticalCodeIds = toListOfStrings(
-      itemRequest.getJsonArray(Item.STATISTICAL_CODE_IDS_KEY));
+    List<String> statisticalCodeIds = getStatisticalCodeIds(itemRequest.getJsonArray(Item.STATISTICAL_CODE_IDS_KEY));
```

Comment:
would that work only for Items? we need to make sure that the same logic is applied for stat codes of Instance and Holdings

Reply:
this PR can be closed. Created PR https://github.com/folio-org/data-import-processing-core/pull/323 with fix  in di-core where the mapping logic is shared for Items, Instances and Holdings